model-customization/data-processing/conversion/docling-conversion.ipynb

```diff
@@ -0,0 +1,228 @@
+    "            pipeline_options=pipeline_options\n",
+    "        )\n",
+    "    }\n",
+    ")\n",
+    "\n",
+    "# Force OCR on the entire page\n",
+    "# pipeline_options = PdfPipelineOptions()\n",
+    "# pipeline_options.do_ocr = True\n",
+    "# pipeline_options.ocr_options.force_full_page_ocr = True\n",
+    "# pipeline_options.ocr_options.lang = [\"en\"]\n",
+    "# pipeline_options.ocr_options = EasyOcrOptions(force_full_page_ocr=True)\n",
+    "# pipeline_options.accelerator_options = AcceleratorOptions(\n",
+    "#     num_threads=4, device=AcceleratorDevice.AUTO\n",
+    "# )\n",
+    "# doc_converter = DocumentConverter(\n",
+    "#     format_options={\n",
+    "#         InputFormat.PDF: PdfFormatOption(\n",
+    "#             pipeline_options=pipeline_options,\n",
+    "#             backend=DoclingParseV4DocumentBackend,\n",
+    "#         )\n",
+    "#     }\n",
+    "# )\n",
+    "\n",
+    "# Use the SmolDocling VLM\n",
+    "# pipeline_options = VlmPipelineOptions()\n",
+    "# pipeline_options.vlm_options = smoldocling_vlm_conversion_options\n",
+    "# doc_converter = DocumentConverter(\n",
+    "#     format_options={\n",
+    "#         InputFormat.PDF: PdfFormatOption(\n",
+    "#             pipeline_options=pipeline_options,\n",
+    "#             pipeline_cls=VlmPipeline,\n",
+    "#         )\n",
+    "#     }\n",
+    "# )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "78ab035e-e05e-41e8-be90-23527b5d4bc4",
+   "metadata": {},
+   "source": [
+    "Finally, we convert every document into Docling JSON as long as it is a valid file type to be converted"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b2a45e16-8fa0-4223-9890-7d75f6869aeb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "\n",
+    "json_files=[]\n",
+    "             \n",
+    "for file in files:\n",
+    "    doc = doc_converter.convert(source=file).document\n",
+    "    doc_dict = doc.export_to_dict()\n",
+    "\n",
+    "    json_output_path = output_dir / f\"{file.stem}.json\"\n",
+    "    with open(json_output_path, \"w\") as f:\n",
+    "        json.dump(doc_dict, f)\n",
+    "        print(f\"Path of JSON output is: {Path(json_output_path).resolve()}\")\n",
+    "        json_files.append(json_output_path.resolve())\n",
+    "\n",
+    "    print(f\"\\nSample:\\n ${doc.export_to_text()[:500]}...\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "868f1a28-e966-4f28-ad2c-f7ba4b0000ae",
+   "metadata": {},
+   "source": [
+    "### Post-Conversion: Illuminator Analysis\n",
+    "\n",
+    "The output of document conversion is not always perfect. Data may become distorted or corrupted, which can negatively affect a model's performance after training. While optional, reviewing your converted data is strongly recommended. The following example explains how to use the Illuminator tool to identify common conversion issues."
```

Comment:
I'm not sure I would call the problems "distorted" or "corrupted" - data distortion or corruption are very strong red flags. Can we find a more accurate and objective way of describing this? Maybe something along the lines of "in the case of documents with complicated authoring, can have table parsing errors or incompletely extracted sections"?